README.md

```diff
@@ -4,3 +4,36 @@ The TrenchBoot website has been converted over to
 [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/). The
 content for the original site is still available under the
 [old-site-archive](https://github.com/TrenchBoot/TrenchBoot.github.io/tree/old-site-archive).
+
+## Local preview
+
+First prepare virtual environment and install dependencies
+
+```shell
+virtualenv -p $(which python3) venv
+source venv/bin/activate
+pip install -r requirements.txt
+```
+
+If you had already done this step then you only need to activate virtual
+environment
+
+```shell
+source venv/bin/activate
+```
+
+To start local server use
+
+```shell
+mkdocs serve
```

Comment:
It would be nice to emphasize the importance of paying attention to the output. It shows useful information, like files that weren't added to `nav` or wrong link formats.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/36/commits/6e3cde13a346bd3cafc1eeb98d7296c199be26d1